### DISCUSSION TITLE
Signaled Seats update

### DISCUSSION THREAD

**Discussion by TishinaVC** [aesefficio/CreateInteriorsMod#26] [Ideas]
Make it so when the seats get a redstone signal they can dismount whatever entity is inside of it.
Useful examples: 

Villagers being transported and offloaded on contraptions/trains/whatever you made
Villagers can go sit down in your restaurant and then after a while leave instead of being trapped for eternity
Hostile entities can be transported in your bay to be dropped on another players base as part of your raid

Hope this is enough of an idea to consider, thank you for your time.

**Comment by a0ax** [comment]
Is a very good idea, but should probably be implemented by Create and not us. We’re just extending their code. Thanks for the idea anyways :)
